static/publications/list_of_publications.html

```diff
@@ -0,0 +1,74 @@
+<!--Here we should include the list of publications with the following format.-->
```

Comment:
This file shouldn't be a hardcoded HTML document! We better come up with a YML database (similar to the members info) that gets rendered to HTML as we like (in case we want to update the style in the future, it'd be MUCH easier!).